code/modules/mod/mod_ai.dm

```diff
@@ -86,3 +87,49 @@
 #undef WEARER_DELAY
 #undef LONE_DELAY
 #undef CELL_PER_STEP
+
+/obj/item/mod/control/proc/ai_fall()
+	if(!wearer)
+		return
+	REMOVE_TRAIT(wearer, TRAIT_FORCED_STANDING, MOD_TRAIT)
+
+/obj/item/mod/ai_minicard
+	name = "AI mini-card"
+	desc = "A small card designed to eject dead AIs. You could use an intellicard to recover it."
+	icon = 'icons/obj/aicards.dmi'
+	icon_state = "minicard"
+	var/mob/living/silicon/ai/stored_ai
+
+/obj/item/mod/ai_minicard/Initialize(mapload, mob/living/silicon/ai/ai)
+	. = ..()
+	if(!ai)
+		return
+	ai.apply_damage(150, BURN)
+	INVOKE_ASYNC(ai, /mob/living/silicon/ai.proc/death)
+	ai.forceMove(src)
+	stored_ai = ai
+	icon_state = "minicard-filled"
```

Comment:
Why does it apply damage to the AI?

Reply:
so it takes damage and takes time to restore instead of instant restoration

Reply:
Because the suit restores it?